ci/gpu/build.sh

```diff
@@ -205,11 +205,11 @@ fi
 
 cd $WORKSPACE/python/cudf
 gpuci_logger "Python py.test for cuDF"
-py.test --cache-clear --basetemp=${WORKSPACE}/cudf-cuda-tmp --junitxml=${WORKSPACE}/junit-cudf.xml -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:${WORKSPACE}/python/cudf/cudf-coverage.xml --cov-report term
+py.test -n 6 --cache-clear --basetemp=${WORKSPACE}/cudf-cuda-tmp --junitxml=${WORKSPACE}/junit-cudf.xml -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:${WORKSPACE}/python/cudf/cudf-coverage.xml --cov-report term
```

Comment:
What's the reasoning behind `6` here? Should we set this programmatically based on something?

Reply:
CI test jobs have 8vCPUS, but only set `PARALLEL_LEVEL` to 4 to have a stable build. But for pytests we can easily use 6 to 7 cores. That was probably the reason I didn't set it programmatically here.

Reply:
@rapidsai/ops any guidance / thoughts here? This looks good to go otherwise.

Reply:
looks good to me.